packages/mongodb-rag-ingest/src/commands/pages.ts

```diff
@@ -53,5 +54,6 @@ export const doPagesCommand = async (
   await updatePages({
     sources,
     pageStore,
+    concurrencyOptions
```

Comment:
rather than passing the whole `concurrencyOptions` object, you should just pass concurrencyOptions.pages`

Reply:
same comment applies for embedding